doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -14,32 +14,28 @@ Prepare your PCB
 
 First, you need to create your PCB for the nRF54H20 SoC.
 
-We highly recommend using the PCB layouts and component values provided by Nordic Semiconductor, especially for clock and power sources.
-However, if you plan to select your own power sources, consider the following limitations:
+We highly recommend using the PCB layouts and component values provided by Nordic Semiconductor, especially for clock and power sources, considering the following limitations:
 
 * The DC/DC inductor must be present on the PCB.
-  If the DC/DC power converter is to be used, the BICR value ``POWER.CONFIG.INDUCTOR`` must also be programmed to ``Present`` (``0``).
-
-  .. note::
-     Currently, there is a bug (LIL-9968) causing the nRF54H20 limited sampling SoC to not wake up from the System OFF if DC/DC is enabled.
-     Temporarily, the workaround is to not use the DC/DC power converter.
-
+* the DC/DC power converter must be present on the PCB and enabled.
+  Also, the ``inductor-present;`` node must be configured either in :file:`zephyr/boards/your_board/nrf54h20dk_bicr.dtsi` or where the section ``bicr: bicr@fff87b0`` is present in the devicetree.
```

Comment:
I am not sure it makes sense for the customer to have any file prefixed with `nrf54h20dk` in their board definition.

Reply:
Good point, suggest to drop the reference to the file completely, something like
Also, the ``inductor-present;`` node must be configured in the section ``bicr: bicr@fff87b0`` in the devicetree.

I guess it somewhere is described how to make your own board specific devicetree, maybe that could be referenced?